acceptance/tests/rototillerTask/command_flags.rb

```diff
@@ -0,0 +1,70 @@
+require 'beaker/hosts'
+require 'rakefile_tools'
+
+test_name 'C97820: can set key/value flag in a RototillerTask' do
+  extend Beaker::Hosts
+  extend RakefileTools
+
+  def create_rakefile_task_segment(flags)
+    segment = ''
+    flags.each do |flag|
+      if flag[:block_syntax]
+        segment += "t.add_flag do |flag|\n"
+        remove_reserved_keys(flag).each do |k, v|
+          segment += "  flag.#{k.to_s} = '#{v}'\n"
+        end
+        segment += "end\n"
+      else
+        segment += "  t.add_flag({"
+        remove_reserved_keys(flag).each do |k, v|
+          segment += ":#{k} => '#{v}',"
+        end
+        segment += "})\n"
+      end
+    end
+    return segment
+  end
+
+  def remove_reserved_keys(h)
+    hash = h.dup
+    [:block_syntax].each do |key|
+      hash.delete(key)
+    end
+    return hash
+  end
+
+  command_flags = [
+      {:name => '--hash-syntax',  :value => 'wow such hash syntax'},
+      {:name => '--us-a-block',   :value => 'wow much block syntax', :block_syntax => true}
+  ]
+
+
+  @task_name    = 'command_flag_testing_key_value'
+  rakefile_contents = <<-EOS
+$LOAD_PATH.unshift('/root/rototiller/lib')
+require 'rototiller'
+
+Rototiller::Task::RototillerTask.define_task :#{@task_name} do |t|
+    #{create_rakefile_task_segment(command_flags)}
+    t.command = 'echo'
```

Comment:
we probably want to use a command that actually takes flags here.  
i also think we should be testing with multiple flags.  lots of existing Rakefiles don't split(' ') on the command args and beaker/others complain about the flags because they are malformed somehow.